gems/sorbet-runtime/lib/types/private/methods/_methods.rb

```diff
@@ -122,15 +132,55 @@ def self.signature_for_method(method)
     @signatures_by_method[key]
   end
 
+  # a module A such that:
+  #   if A is included or extended by a module B,
+  #   then we want B to have the hooks installed on itself
+  # should extend this module.
+  module VirallyInstallHooks
+    def included(arg)
+      super(arg)
+      if arg.is_a?(Module)
+        ::T::Private::Methods.install_hooks(arg)
+      end
+    end
+
+    def extended(arg)
+      super(arg)
+      if arg.is_a?(Module)
```

Comment:
Is it not necessarily the case that `arg` is a module?

Reply:
I think this might be a remnant from when I was playing around with different implementation strategies. Removing this seems to have no ill effects.